language-support/ts/daml-react/hooks.test.ts

```diff
@@ -26,8 +30,37 @@ jest.mock('@daml/ledger', () => class {
   fetchByKey(...args: unknown[]): Promise<string> {
     return mockFetchByKey(...args);
   }
+
+  streamQuery(...args: unknown[]): Stream<object, string, string, string>{
+    return mockStreamQuery(...args);
+  }
+
+  streamFetchByKey(...args: unknown[]): Stream<object, string, string, string>{
+    return mockStreamFetchByKey(...args);
+  }
 });
 
+/**
+ * Returns a mock stream object using an `EventEmitter` to implement on, off functions.
+ */
+const mockStream = (): [Stream <string[], string, string, string>, EventEmitter] =>
+{
+  const emitter = new EventEmitter();
+  const stream =
+    {
+      on: (type: string, listener: any) => emitter.on(type, listener),
+      off: (type: string, listener: any) => emitter.off(type, listener),
```

Comment:
To be somewhat closer to reality and make the linter shut up (at least in my editor):
```suggestion
      // eslint-disable-next-line @typescript-eslint/no-explicit-any
      on: (type: string, listener: (...args: any[]) => void) => emitter.on(type, listener),
      // eslint-disable-next-line @typescript-eslint/no-explicit-any
      off: (type: string, listener: (...args: any[]) => void) => emitter.on(type, listener),
```
I'm actually surprised our linting test has not complained about the `any`. Can you please find out if the linter is really running?